domain-layer/submission/submission.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+
+package topcoder.domain.submission;
+
+message Submission {
+  string id = 1;
+  string challengeId = 2;
```

Comment:
needs to be snake_case. It might be a good idea to install a protoc linter. I have `.vscode` checked with in with a `settings.json` config. If you have clang-format `xaver.clang-format` installed you should see lint errors.

Reply:
See here: https://developers.google.com/protocol-buffers/docs/style